tests/dummy/public/test-forms/basic-fields/form.ttl

```diff
@@ -262,6 +302,17 @@ ext:conceptSchemeMultiSelectorField a form:Field ;
 
 ext:mainFg form:hasField ext:conceptSchemeMultiSelectorField .
 
+ext:conceptSchemeMultiSelectorWithOptionPredicatesField a form:Field ;
+    sh:name "Multiselect with form:options as separate predicates" ;
+    sh:order 280 ;
+    sh:path ext:conceptSchemeMultiSelectorValue ;
+    ext:conceptScheme exampleConceptSchemes:foo-bar-baz ;
+    ext:searchEnabled false ; # This is not even working with the json options for this component
```

Comment:
What doesn't work exactly? I tested it again in Firefox, and setting this to false does seem to disable the search field? There is at least a difference compared to setting this to `true` (or removing it, since true is the default).